### PR TITLE
Chore(SSM): Add AWS SSM Chaos Docs

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit <udit@chaosnative.com>
**What this PR does / why we need it**:
- Add AWS SSM Chaos Docs
- Update kube aws docs
- Update sidebar for vmware
- Add embedmd run

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for [DCO](https://github.com/litmuschaos/litmus-docs/blob/master/CONTRIBUTING.md#sign-your-work) check to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: docs/aws-ssm-chaos-by-id.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+  <tr>
+    <th> Variables </th>
+    <th> Description </th>
+    <th> Specify In ChaosEngine </th>
+    <th> Notes </th>
+  </tr>
+  <tr> 
+    <td> EC2_INSTANCE_ID </td>
+    <td> Instance ID of the target ec2 instance. Multiple IDs can also be provided as a comma(,) separated values</td>
+    <td> Optional </td>
```
Comment (ispeakc0de) [inline docs/aws-ssm-chaos-by-id.md]:
```suggestion
    <td> Mandatory </td>
```

File: docs/aws-ssm-chaos-by-id.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+    <td> Multiple IDs can be provided as `id1,id2` </td>
+  </tr>
+  <tr> 
+    <td> TOTAL_CHAOS_DURATION </td>
+    <td> The total time duration for chaos insertion (sec) </td>
+    <td> Optional </td>
+    <td> Defaults to 30s </td>
```
Comment (ispeakc0de) [inline docs/aws-ssm-chaos-by-id.md]:
```suggestion
    <td> Defaults to 60s </td>
```

File: docs/aws-ssm-chaos-by-id.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+  </tr>
+  <tr> 
+    <td> CHAOS_INTERVAL </td>
+    <td> The interval (in sec) between successive instance termination.</td>
+    <td> Optional </td>
+    <td> Defaults to 30s </td>
```
Comment (ispeakc0de) [inline docs/aws-ssm-chaos-by-id.md]:
```suggestion
    <td> Defaults to 60s </td>
```

File: docs/aws-ssm-chaos-by-tag.md [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+
+<table>
+  <tr>
+    <th> Variables </th>
+    <th> Description </th>
+    <th> Specify In ChaosEngine </th>
+    <th> Notes </th>
+  </tr>
+  <tr> 
+    <td> EC2_INSTANCE_ID </td>
```
Comment (ispeakc0de) [inline docs/aws-ssm-chaos-by-tag.md]: do we need this?

**Review by ispeakc0de — DISMISSED** [review]

**Review by ispeakc0de — DISMISSED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**